jscomp/others/js_json.ml

```diff
@@ -60,6 +61,31 @@ let test (type a) (x : 'a) (v : a kind) : bool =
   | Array -> Js_array.isArray x 
   | Object -> (Obj.magic x) != Js.null && Js.typeof x = "object" && not (Js_array.isArray x )
 
-
 external parse : string -> t = "JSON.parse" [@@bs.val]
 (* TODO: more docs when parse error happens or stringify non-stringfy value *)
+
+let null : t = Obj.magic Js.null 
+
+let string s : t = Obj.magic s
+
+let number f : t = Obj.magic f
+
+let numberOfInt i : t = Obj.magic i
+
+let boolean b : t = Obj.magic (Js_boolean.to_js_boolean b)
```

Comment:
@bobzhang In my case I really to have a boolean value in the JSON.

This module goal is to provide api to read and create any JSON values not just serializing OCaml values. JSON contains boolean we must provide a way to write a boolean. 

We could also provide an API like `boolAsInt` which would do what you describe. It's then up to the caller to decide what to use.